Source/WebCore/css/StyleProperties.cpp

```diff
@@ -383,14 +383,14 @@ String StyleProperties::getPropertyValue(CSSPropertyID propertyID, Document* doc
     }
 }
 
-std::optional<Color> StyleProperties::propertyAsColor(CSSPropertyID property) const
+std::optional<StyleColor> StyleProperties::propertyAsColor(CSSPropertyID property) const
 {
     auto colorValue = getPropertyCSSValue(property);
     if (!is<CSSPrimitiveValue>(colorValue))
         return std::nullopt;
 
     auto& primitiveColor = downcast<CSSPrimitiveValue>(*colorValue);
-    return primitiveColor.isRGBColor() ? primitiveColor.color() : CSSParser::parseColorWithoutContext(colorValue->cssText());
+    return primitiveColor.isRGBColor() ? primitiveColor.color() : StyleColor { CSSParser::parseColorWithoutContext(colorValue->cssText()) };
```

Comment:
I think parseColorWithoutContext() returns a StyleColor, so the wrapping is unneeded.